### PR TITLE
Add instant availability check for DebridLink 

### PR DESCRIPTION
**Pull request by MrMonkey42**
Closes https://github.com/mhdzumair/MediaFusion/issues/23

Tested the functionality locally - https://github.com/mhdzumair/MediaFusion/assets/147980138/72be4555-7f65-4057-87c9-d88b6bc35c1a

### CODE REVIEW

**Comment by mhdzumair** [comment]
Thanks @MrMonkey42 for your contribution. i was not able to implement this feature because of not having the subscription to test.  

if possible could you also work on the https://github.com/mhdzumair/MediaFusion/issues/30

File: utils/parser.py [hunk 1]
```diff
@@ -7,9 +7,8 @@
 from db.config import settings
 from db.models import Streams, TVStreams
 from db.schemas import Stream, UserData
-from streaming_providers.realdebrid.utils import (
-    order_streams_by_instant_availability_and_date,
-)
+import streaming_providers.realdebrid.utils as RealDebridUtils
+import streaming_providers.debridlink.utils as DebridLinkUtils
```
Comment (mhdzumair) [inline utils/parser.py:11]:
```suggestion
from streaming_providers.realdebrid import utils as rd_utils
from streaming_providers.debridlink import utils as dl_utils
```

File: utils/parser.py [hunk 2]
```diff
@@ -35,7 +34,12 @@ def parse_stream_data(
         user_data.streaming_provider
         and user_data.streaming_provider.service == "realdebrid"
     ):
-        streams = order_streams_by_instant_availability_and_date(streams, user_data)
+        streams = RealDebridUtils.order_streams_by_instant_availability_and_date(streams, user_data)
+    elif (
+        user_data.streaming_provider
+        and user_data.streaming_provider.service == "debridlink"
+    ):
+        streams = DebridLinkUtils.order_streams_by_instant_availability_and_date(streams, user_data)
```
Comment (mhdzumair) [inline utils/parser.py:42]:
```suggestion
        streams = rd_utils.order_streams_by_instant_availability_and_date(streams, user_data)
    elif (
        user_data.streaming_provider
        and user_data.streaming_provider.service == "debridlink"
    ):
        streams = dl_utils.order_streams_by_instant_availability_and_date(streams, user_data)
```

**Review by mhdzumair — CHANGES_REQUESTED** [review]

File: utils/parser.py (hunk 1 above)
Comment (MrMonkey42) [reply utils/parser.py:11]: I don't have much development experience with Python. Curious what is the difference between those import ways?

Comment (mhdzumair) [reply utils/parser.py:11]: in the above imports statements are doing the same. Importing utils namespace into current file. The `from` statement is specially used to specifying that is importing from submodule contents. Again, this is something I've been practiced from pythonic world and my preference. 

The main reason for the changes was to follow python variable guide as snake case. Such as `RealDebridUtils` -> `rd_utils`. `rd` is short name.

**Review by mhdzumair — APPROVED** [review]

**Pull request merged**